NixOS build broken

https://github.com/nbfc-linux/nbfc-linux/commit/7feb52631ccd5be51a9e702a7a4c5a0e7086389e#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L6 breaks the build on NixOS.

When I run `sudo nixos-rebuild switch --flake .#`, I get the following build logs
```
error: builder for '/nix/store/yva6mxlzl0k6hb0747gwfciyg9s63sw7-nbfc-linux.drv' failed with exit code 2;
       last 10 log lines:
       > sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/zsh/_nbfc.in > completion/zsh/_nbfc
       > sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/zsh/_nbfc_service.in > completion/zsh/_nbfc_service
       > installing
       > install -Dm 755 src/nbfc_service  /nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin/nbfc_service
       > install -Dm 755 src/ec_probe      /nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin/ec_probe
       > install -Dm 755 src/nbfc          /nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin/nbfc
       > # /usr/local/etc/nbfc
       > mkdir -p /etc/nbfc
       > mkdir: cannot create directory '/etc/nbfc': Permission denied
       > make: *** [Makefile:116: install-configs] Error 1
       For full logs, run 'nix log /nix/store/yva6mxlzl0k6hb0747gwfciyg9s63sw7-nbfc-linux.drv'.
error: 1 dependencies of derivation '/nix/store/kiwninfsvp0hr5j1aldi7j2wqlimcw2f-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/bb3dgkx847cihy858aq16mb7s1a2b8v3-unit-script-nbfc_service-start.drv' failed to build
e
```

`nix log /nix/store/yva6mxlzl0k6hb0747gwfciyg9s63sw7-nbfc-linux.drv` yields
```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/5mwb239v480klgjrg1pyra7gg6hzzfjz-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: SHELL=/nix/store/rhvbjmcfnkg8i2dxpzr114cp1ws7f667-bash-5.2-p15/bin/bash PREFIX=\$\(out\) confdir=/etc

=======================================================================
This Makefile is deprecated. Use ./configure for future builds!
=======================================================================

gcc -DNDEBUG -DSYSCONFDIR=\"/etc\" -DDATADIR=\"/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share\" -DRUNSTATEDIR=\"/var/run\" -DVERSION=\"0.1.15\" -Wall -Os -flto src/build.c -o src/nbfc_service -lm -s
gcc -DNDEBUG -DSYSCONFDIR=\"/etc\" -DDATADIR=\"/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share\" -DRUNSTATEDIR=\"/var/run\" -DVERSION=\"0.1.15\" -Wall -Os -flto src/client.c -o src/nbfc -lm -s
gcc -DNDEBUG -DSYSCONFDIR=\"/etc\" -DDATADIR=\"/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share\" -DRUNSTATEDIR=\"/var/run\" -DVERSION=\"0.1.15\" -Wall -Os -flto src/ec_probe.c -o src/ec_probe -lm -s
[01m[Ksrc/ec_probe.c:[m[K In function '[01m[KRegister_LoadDump[m[K':
[01m[Ksrc/ec_probe.c:606:3:[m[K [01;35m[Kwarning: [m[Kignoring return value of '[01m[Kfread[m[K' declared with attribute '[01m[Kwarn_unused_result[m[K' [[01;35m[K]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result-Wunused-result]8;;[m[K]
  606 |   [01;35m[Kfread(header, 1, sizeof(RegisterHeader) - 1, fh)[m[K;
      |   [01;35m[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[m[K
[01m[Ksrc/ec_probe.c:626:7:[m[K [01;35m[Kwarning: [m[Kignoring return value of '[01m[Kfread[m[K' declared with attribute '[01m[Kwarn_unused_result[m[K' [[01;35m[K]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result-Wunused-result]8;;[m[K]
  626 |       [01;35m[Kfread(hex, 1, 1, fh)[m[K;
      |       [01;35m[K^~~~~~~~~~~~~~~~~~~~[m[K
gcc -DNDEBUG -DSYSCONFDIR=\"/etc\" -DDATADIR=\"/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share\" -DRUNSTATEDIR=\"/var/run\" -DVERSION=\"0.1.15\" -Wall -Os -flto src/test_model_config.c -o src/test_model_config -lm -s
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < doc/ec_probe.1.in > doc/ec_probe.1
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < doc/nbfc.1.in > doc/nbfc.1
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < doc/nbfc_service.1.in > doc/nbfc_service.1
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < doc/nbfc_service.json.5.in > doc/nbfc_service.json.5
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < etc/systemd/system/nbfc_service.service.in > etc/systemd/system/nbfc_service.service
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < etc/init.d/nbfc_service.openrc.in > etc/init.d/nbfc_service.openrc
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < etc/init.d/nbfc_service.systemv.in > etc/init.d/nbfc_service.systemv
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/bash/ec_probe.in > completion/bash/ec_probe
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/bash/nbfc.in > completion/bash/nbfc
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/bash/nbfc_service.in > completion/bash/nbfc_service
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/fish/ec_probe.fish.in > completion/fish/ec_probe.fish
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/fish/nbfc.fish.in > completion/fish/nbfc.fish
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/fish/nbfc_service.fish.in > completion/fish/nbfc_service.fish
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/zsh/_ec_probe.in > completion/zsh/_ec_probe
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/zsh/_nbfc.in > completion/zsh/_nbfc
sed -e 's|@BINDIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin|g' -e 's|@DATADIR@|/nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/share|g' -e 's|@SYSCONFDIR@|/etc|g' -e 's|@RUNSTATEDIR@|/var/run|g' -e 's|@VERSION@|0.1.15|g' < completion/zsh/_nbfc_service.in > completion/zsh/_nbfc_service
@nix { "action": "setPhase", "phase": "installPhase" }
installing
install -Dm 755 src/nbfc_service  /nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin/nbfc_service
install -Dm 755 src/ec_probe      /nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin/ec_probe
install -Dm 755 src/nbfc          /nix/store/yzjy2h8qyc25p088avhnnw2gk7gp5iha-nbfc-linux/bin/nbfc
# /usr/local/etc/nbfc
mkdir -p /etc/nbfc
mkdir: cannot create directory '/etc/nbfc': Permission denied
make: *** [Makefile:116: install-configs] Error 1

```

The build fails to create directory `/etc/nbfc`. Previously, the `PREFIX` would point to the directory of the derivation in the nix store (see also https://github.com/nbfc-linux/nbfc-linux/blob/main/flake.nix#L17).

--- a/README.md
+++ b/README.md
@@ -30,10 +30,9 @@ Installation
   - Either via AUR (`yaourt -S nbfc-linux`)
   - Or by using the PKGBUILD [nbfc-linux-git](pkgbuilds/nbfc-linux-git/PKGBUILD)
 
-- With Nix:
-  - With legacy nix (`nix-env -if .`)
-  - With nix flakes (`nix profile install`)
-  - [On NixOS](nixos-installation.md)
+- NixOS:
+  - [Declaratively](nixos-installation-new.md)(with nix flakes)
+  - [Imperatively](nixos-installation.md) (with `nix profile install` or `nix-env -if .`)
 
 - Fedora - this will not build the current working directory, but downloads a source archive from chosen release on github and builds an installable rpm:
   - ensure, that podman is installed

--- a/default.nix
+++ b/default.nix
@@ -1,7 +1,9 @@
 (import (
-  fetchTarball {
-    url = "https://github.com/edolstra/flake-compat/archive/99f1c2157fba4bfe6211a321fd0ee43199025dbf.tar.gz";
-    sha256 = "0x2jn3vrawwv9xp15674wjz9pixwjyj3j771izayl962zziivbx2"; }
-) {
-  src =  ./.;
-}).defaultNix
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/99f1c2157fba4bfe6211a321fd0ee43199025dbf.tar.gz";
+      sha256 = "0x2jn3vrawwv9xp15674wjz9pixwjyj3j771izayl962zziivbx2";
+    }
+  ) {
+    src = ./.;
+  })
+.defaultNix

--- a/flake.nix
+++ b/flake.nix
@@ -1,43 +1,34 @@
 {
   inputs = {
-    nixpkgs.url = github:nixOS/nixpkgs/release-23.05;
-    utils.url = github:numtide/flake-utils;
+    nixpkgs.url = "github:nixOS/nixpkgs/release-23.05";
+    utils.url = "github:numtide/flake-utils";
   };
-  outputs = { nixpkgs, utils, ... }:
-    utils.lib.eachSystem [ "i686-linux" "x86_64-linux" ]
-      (system:
-        let
-          pkgs = import nixpkgs {
-            inherit system;
-          };
-        in
-        rec {
-          packages =
-            let
-              buildFlags = [ "PREFIX=$(out)" "confdir=/etc" ];
-            in
-            rec {
-              nbfc = pkgs.stdenv.mkDerivation {
-                inherit buildFlags;
+  outputs = {
+    nixpkgs,
+    utils,
+    ...
+  }:
+    utils.lib.eachSystem ["i686-linux" "x86_64-linux"]
+    (system: let
+      pkgs = nixpkgs.legacyPackages.${system};
+    in rec {
+      packages = {
+        nbfc = pkgs.stdenv.mkDerivation {
+          name = "nbfc-linux";
+          version = "0.1.15";
 
-                name = "nbfc-linux";
-                version = "0.1.6";
+          src = nixpkgs.lib.cleanSource ./.;
 
-                src = nixpkgs.lib.cleanSource ./.;
-
-                installPhase =
-                  let
-                    installFlags = [ "PREFIX=$out" ];
-                  in
-                  ''
-                    make ${builtins.concatStringsSep " " installFlags}\
-                         install-core \
-                         install-configs\
-                         install-docs\
-                         install-completion
-                  '';
-              };
-            };
-          defaultPackage = packages.nbfc;
-        });
+          nativeBuildInputs = with pkgs; [
+            autoreconfHook
+          ];
+          configureFlags = [
+            "--prefix=${placeholder "out"}"
+            "--sysconfdir=${placeholder "out"}/etc"
+            "--bindir=${placeholder "out"}/bin"
+          ];
+        };
+        default = packages.nbfc;
+      };
+    });
 }

--- a/nixos-installation-new.md
+++ b/nixos-installation-new.md
@@ -1,0 +1,93 @@
+# Installing on NixOS
+
+Flakes must be enabled to follow this guide, if you do not want to enable flakes check [this installation guide](./nixos-installation.md)
+
+
+- Add this flake to inputs in your configuration:
+
+```nix
+# flake.nix
+{
+  inputs = {
+    # ...
+    nbfc-linux = {
+      url = "github:nbfc-linux/nbfc-linux";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    # ...
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+    home-manager,
+    ...
+  } @ inputs: {
+    # ...
+   };
+  };
+}
+
+```
+
+NOTE: Check status of [nbfc-linux on nixpkgs](https://github.com/NixOS/nixpkgs/pull/296199) , if it has been merged into nixpkgs no need to use flakes simply add nbfc-linux to envrioment.systemPackages 
+
+- Mention it in the packages + enable nbfc service
+```nix
+# nbfc.nix
+{
+  config,
+  inputs,
+  pkgs,
+  ...
+}: let
+  myUser = "gaurav"; #adjust this to your username
+  command = "bin/nbfc_service --config-file '/home/${myUser}/.config/nbfc.json'";
+
+in {
+  environment.systemPackages = with pkgs; [
+    inputs.nbfc-linux.packages.x86_64-linux.default
+  ];
+  systemd.services.nbfc_service = {
+    enable = true;
+    description = "NoteBook FanControl service";
+    serviceConfig.Type = "simple";
+    path = [pkgs.kmod];
+    script = "${inputs.nbfc-linux.packages.x86_64-linux.default}/${command}";
+    wantedBy = ["multi-user.target"];
+  };
+}
+```
+
+- Import nbfc.nix into configuration.nix
+```nix
+# configuration.nix
+ imports = [
+     # ...
+    ./nbfc.nix
+  ];
+```
+
+# Chose configuration file
+
+Now if you try to set config the normal way you will encounter the following issue
+```bash
+$ sudo nbfc config --set "Asus Zenbook Flip UX360"
+Error: [Errno 30] Read-only file system: '/nix/store/is476j7dp9ggy8rrqcmx68dpj21n3v0f-nbfc-linux/etc/nbfc/nbfc.json'
+```
+
+Solution:
+```json
+{"SelectedConfigId": "<your notebook>"}
+```
+Write this to `~/.config/nbfc.json` (or your location of choice)
+
+Insert the [file name of the config file](https://github.com/nbfc-linux/nbfc-linux/tree/main/share/nbfc/configs) (without ".json"), *not* the `NotebookModel` value inside that config file.
+
+example:
+```json
+{"SelectedConfigId": "Asus Zenbook Flip UX360UAK"}
+```
+
+This usage of `--config-file` option of nbfc may also help on similar distros. 
+

--- a/share/nbfc/configs/HP Pavilion dk15.json
+++ b/share/nbfc/configs/HP Pavilion dk15.json
@@ -1,0 +1,77 @@
+{
+ "NotebookModel": "HP Pavilion dk15",
+ "Author": "Celibistrial",
+ "EcPollInterval": 1000,
+ "ReadWriteWords": false,
+ "CriticalTemperature": 85,
+ "FanConfigurations": [
+	{
+	 "ReadRegister": 17,
+	 "WriteRegister": 20,
+	 "MinSpeedValue": 19,
+	 "MaxSpeedValue": 53,
+	 "IndependentReadMinMaxValues": false,
+	 "MinSpeedValueRead": 0,
+	 "MaxSpeedValueRead": 0,
+	 "ResetRequired": false,
+	 "FanSpeedResetValue": 0,
+	 "FanDisplayName": "CPU Fan",
+	 "TemperatureThresholds": [
+	  {
+	   "UpThreshold": 40,
+	   "DownThreshold": 0,
+	   "FanSpeed": 00.0
+	  },
+	  {
+	   "UpThreshold": 50,
+	   "DownThreshold": 30,
+	   "FanSpeed": 20.0
+	  },
+	  {
+	   "UpThreshold": 60,
+	   "DownThreshold": 40,
+	   "FanSpeed": 25.0
+	  },
+	  {
+	   "UpThreshold": 70,
+	   "DownThreshold": 50,
+	   "FanSpeed": 47.5
+	  },
+	  {
+	   "UpThreshold": 80,
+	   "DownThreshold": 60,
+	   "FanSpeed": 77.5
+	  },
+	  {
+	   "UpThreshold": 85,
+	   "DownThreshold": 70,
+	   "FanSpeed": 100.0
+	  }
+	 ],
+	 "FanSpeedPercentageOverrides": [
+	  {
+	   "FanSpeedPercentage": 76.92308,
+	   "FanSpeedValue": 39,
+	   "TargetOperation": "ReadWrite"
+	  },
+	  {
+	   "FanSpeedPercentage": 38.46154,
+	   "FanSpeedValue": 34,
+	   "TargetOperation": "ReadWrite"
+	  }
+	 ]
+	}
+ ],
+ "RegisterWriteConfigurations": [
+	{
+	 "WriteMode": "Or",
+	 "WriteOccasion": "OnInitialization",
+	 "Register": 15,
+	 "Value": 8,
+	 "ResetRequired": true,
+	 "ResetValue": 5,
+	 "ResetWriteMode": "Set",
+	 "Description": "Override"
+	}
+ ]
+}

